ctagsdotd/kotlin.ctags

```diff
@@ -1,12 +0,0 @@
---langdef=kotlin
```

Comment:
TODO: See if the built-in parser is less powerful than this. If so we can enhance it with what it's missing.